src/pattern-library/components/patterns/MoleculePatterns.js

```diff
@@ -98,6 +101,58 @@ export default function MoleculePatterns() {
           </PatternExample>
         </PatternExamples>
       </Pattern>
+
+      <Pattern title="Modal container">
+        <p>
+          The <code>modal</code> pattern positions and sizes a modal container
+          appropriately based on viewport size.
+        </p>
+        <PatternExamples>
+          <PatternExample details="Responsive Modal container positioning and sizing">
+            <div>
+              <LabeledButton
+                variant="primary"
+                onClick={() => setShowExample1(true)}
+              >
+                Show example
+              </LabeledButton>
+              <div
+                className="hyp-u-overlay"
+                style={
+                  showExample1 ? 'visibility:visible' : 'visibility:hidden'
+                }
```

Comment:
```suggestion
style={ visibility: showExample1 ? 'visible' : 'hidden' }
```

The use of strings as values for the `style` prop in Preact is technically supported, although I consider it preferable to pass an object of CSS property names and values instead (as if accessing the `element.style` property). React only supports object values as a form of XSS protection (eg. that could happen if using user input in the style string).

Reply:
Roger.